micro-ui/web/micro-ui-internals/packages/react-components/src/hoc/ViewComposer/index.js

```
@@ -164,20 +164,24 @@ const ViewComposer = ({ isLoading = false, data, cardErrors, ...props }) => {
-              })}
-            </Card>
+            noCardStyle ?
+              <div>
+                {sections?.map((section, sectionIdx) => {
+                  return renderCardSectionJSX(section, cardErrors?.[card?.errorName ? card?.errorName : card?.name]);
+                })}
+              </div> :
+              <Card
+                style={activeNav && card.navigationKey ? (activeNav !== card.navigationKey ? { display: "none" } : {}) : {}}
+                className={`employeeCard-override ${card?.cardStyle ? card?.cardStyle : ""} ${hasErrors ? "card-error" : ""}`}
+                ReactRef={hasErrors ? (el) => (cardRefs.current[cardIdx] = el) : null}>
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)